packages/@o3r/rules-engine/package.json

```diff
@@ -37,6 +37,7 @@
     "globby": "^11.1.0",
     "jsonpath-plus": "^7.0.0",
     "rxjs": "^7.4.0",
+    "typescript": "~4.9.4",
```

Comment:
__OPEN QUESTION__ For the dependencies used inside the schematics/builders of a package, should we turn them as optional ( we can talk here about @o3r/schematics which is used only by the schematics scripts on all packages, or typescript (api) which is used in builders/schematic scripts on the majority of the packages ?

Reply:
I would say that if the dependency is required only by builder and/or schematics, this should be `Optional PeerDependency` (or `dependencies` in case of `core` package).